sentry-sidekiq/lib/sentry/sidekiq/util.rb

```diff
@@ -0,0 +1,21 @@
+module Sentry
+  module Sidekiq
+    class Util
+      SIDEKIQ_NAME = "Sidekiq".freeze
+
+      def self.transaction_name_from_context(context)
```

Comment:
I'm thinking maybe this can be moved into `ContextFilter` as well? it seems reasonable to be able to extract the transaction name as well. wdyt?